src/components/MyGames.js

```diff
@@ -70,6 +70,7 @@ const MyGames = ({ userGames, addGame, removeGame }) => {
                     bottom: 0,
                   }}
                 ></Image>
+                <Text style={styles.gameTitle}>{itemData.item.game_title}</Text>
```

Comment:
This makes our lack of cover images way more bearable lol. Awesome idea, Tom. I will apply this to the `MyGames` and `SearchGames` screens.